### DISCUSSION TITLE
I created a raspberry pi image

### DISCUSSION THREAD

**Discussion by boehm-e** [explosion/spaCy#2889] [Show & Tell]
Hi, 
if anyone is interested, I create a Raspberry Pi image with 
spacy, en_core_web_sm and en_coref_sm (from neuralcoref)  installed:

to download it : https://drive.google.com/file/d/1swM9xNzCGpFat2o3iK_CiM8tm4rhFwj8/view?usp=sharing

The image is based on https://github.com/bamarni/pi64/ , a 64bit linux for raspberry pi :
currently tested on PI 3B+ (2017) but should work on all raspberry pi 3.

**Comment by ines** [comment]
@boehm-e Thanks, this is really cool! In case you haven't seen it yet, we shared it on Twitter: https://twitter.com/spacy_io/status/1058708288643260416 🎉

Would you be interested in sharing the image via a separare repo, GitHub page etc.? We'd love to include it in the [spaCy universe](https://spacy.io/universe).

I'd also be very curious to hear about what you've been building with it and all (potential) cool things that could be done with spaCy on a Raspberry Pi!

**Comment by boehm-e** [comment]
@ines 
Thank you for sharing :+1: 

As you sugested, i created a repo for this project at : https://github.com/boehm-e/spacy-raspberry

And as you guessed, i use spacy with speech recognition on raspberry pi, in order to animate some characters with speech : 

![demo](https://user-images.githubusercontent.com/11290215/48054257-23bd4b00-e1ad-11e8-8f38-7461d045aa8e.gif)
